.docker/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ghcr.io/thomasonzhou/ros-humble-dev
```

Comment:
What is the content of this?
Could you share `Dockerfile` of this image?

Reply:
Thanks.

We may want to transfer https://github.com/thomasonzhou/ros2_anywhere to https://github.com/reazon-research/ but it's out of scope of this PR.